raster/r.kappa/calc_metrics.c

```diff
@@ -142,10 +148,12 @@ void calc_metrics(void)
         else
             metrics->producers_accuracy[i] = 100 * (pii[i] / pj[i]);
         /* theta 1 */
-        p0 += pii[i];
+        update_sum(&p0, &p0c, pii[i]);
         /* theta 2 */
-        pC += pi[i] * pj[i];
+        update_sum(&pC, &pCc, pi[i] * pj[i]);
     }
+    p0 = p0 + p0c;
+    pC = pC + pCc;
```

Comment:
`+=` expresses the intention better in C (and Python). Same for inter1.

Reply:
I think we should move update_sum to main library (gmath) and replace most of += with nkb in all libraries and modules. But that's for another day.

Reply:
I agree that's best.

Here, I meant just:
 
```suggestion
    p0 += p0c;
    pC += pCc;
```

Sorry, I was not clear.

Reply:
Is there a some kind of style guideline demanding such change? I do use += for counters and accumulators in loops but prefer to have all terms explicit when summation is neither of them.